readme.md

```diff
@@ -24,19 +24,44 @@ This will allow staff to modify a members benefits during the current membership
 CiviCRM has support for many payment processors, including several Direct Debit payment processors. With these “online” payment processors, when the membership comes to renew, the logic is actually managed by the payment processor in order to renew the membership and take next years payment. CiviCRM doesn’t however have any functionality for memberships where the payment is “offline” i.e. some Direct Debit processes or where you invoice clients in advance of receiving the payment. With Membership extras CiviCRM now fully supports offline automated renewal including sending email notifications with invoices for payment. We also have created a new offline batch direct debit export module which allows for full management of high volume direct debits through export processes.
 
 ## How do I get Membership Extras?
-Membership Extras is designed to work with CiviCRM 4.7.x or 5.x plus. If you are on an earlier version of CiviCRM, you will need to upgrade your site first or contact info@compucorp.co.uk if you needs assistance to do so.
+Membership Extras is designed to work with a patched version CiviCRM 5.19.4 plus. If you are on an earlier version of CiviCRM, you will need to upgrade your site first or contact info@compucorp.co.uk if you need assistance to do so.
```

Comment:
we are currently testing on v5.24.4 , and it is the version we are supporting for next release , so I don't think  we need to include the others since there is no guarantee that they will still work.